source/extensions/filters/http/composite/filter.h

```diff
@@ -25,6 +30,28 @@ struct FilterStats {
   ALL_COMPOSITE_FILTER_STATS(GENERATE_COUNTER_STRUCT)
 };
 
+class MatchedActionInfoType : public StreamInfo::FilterState::Object {
```

Comment:
tiny nit: I personally feel `MatchedActionInfo` is a better name as I am not sure what does`Type` refer to?

WDYT?